Mixpanel/Variant.swift

```diff
@@ -157,7 +157,7 @@ class Variant: NSObject, NSCoding {
         if object === self {
             return true
         } else {
-            return self.ID == object.ID
+            return self.ID == object.ID && self.actions == object.actions
```

Comment:
it looks like VariantAction's equal check only checks the name. is that enough to check if they're actually doing the same thing? i.e. if someone changes an action in their ab test will the name that's sent back change too?

also do we want to do this on a separate branch to fix it on master earlier than september?